examples/benchmark/README.md

```diff
@@ -0,0 +1,40 @@
+## Clickstream benchmark toole
+
+We have developed a tool for performance tests, which utilizes ECS Fargate to continuously send test data to Clickstream. The number of ECS Service tasks can be increased or decreased to adjust the RPS (Requests per Second).
+
+Follow below step to start your Clickstream performance test
+1. Add configuration file:
+
+Please download your `amplifyconfiguration.json` file from your web console of the solution, then copy it to the root
+folder which at the same level as README file. then you can execute the following command to create and send your sample
+data.
+
```

Comment:
add prerequesisities, for example,

- Python 3.xx
- Docker engine 
- configure `awscli` with proper privileges
- ...

Reply:
Added